tests/bluetooth/bsim_bt/bsim_test_advx/src/main.c

```diff
@@ -211,6 +211,8 @@ static void test_advx_main(void)
 
 	k_sleep(K_MSEC(400));
 
+	/* FIXME: re-enable of 251 byte advertising PDU support is added */
```

Comment:
I guess you mean "if" and not "of", however I think "when" would be most appropriate :)

Reply:
done